packages/validator/src/cli-validator/run-validator.js

```diff
@@ -310,16 +309,10 @@ const processInput = async function(program) {
     // run validator, print the results, and determine if validator passed
     let results;
     try {
-      results = validator(
-        logger,
-        swagger,
-        configObject,
-        spectralResults,
-        debug
-      );
+      results = validator(logger, swagger, configObject, spectralResults);
     } catch (err) {
       logError(chalk, 'There was a problem with a validator.', getError(err));
-      if (debug) {
+      if (verbose) {
```

Comment:
The more I think about it, the more I like the idea of doing away with the verbose flag and using the log levels to distinguish these invocations but I think we can discuss that and do it in a separate PR because there are some other implications. Wanted to bring it up so I wouldn't forget but fine to leave it as is right now 👍